pyuavcan/transport/can/media/pythoncan/_pythoncan.py

```diff
@@ -85,6 +85,9 @@ def __init__(
 
             - Interface ``usb2can`` is described in https://python-can.readthedocs.io/en/stable/interfaces/usb2can.html.
               Example: ``usb2can:ED000100``
+              
+            - Interface ``seeedstudio`` is described in https://python-can.readthedocs.io/en/3.3.2/interfaces/seeedstudio.html.
```

Comment:
How come it's available in v3.3.2, unavailable in v3.3.3/3.3.4, and then available again in v4.0? Is there some story behind it?

Reply:
I have no idea. I am just using it. At the moment it works.

Reply:
I think the link should point to v4.0 instead of v3.3.2.